system/scripts/zone/core/calc_skill.cs

```diff
@@ -337,6 +389,8 @@ public float SCR_GET_ShootTime(Skill skill)
 		var sklSpdRate = skill.Properties.GetFloat(PropertyName.SklSpdRate);
 		var defShootTime = skill.Data.ShootTime.TotalMilliseconds;
 
+		if (skill.IsMonsterSkill) sklSpdRate *= ZoneServer.Instance.Conf.World.MonsterSkillSpeed;
```

Comment:
I'm going to remove this for now, under the assumption that we don't really need to modify the shoot time. It's already being affected by the the speed rate, which we are also modifying with the conf option, and when I tested it, setting this didn't seem to do anything.